pbm/client.go

```diff
@@ -94,3 +97,121 @@ func (c *Client) CheckRequirements(ctx context.Context, hubs []types.PbmPlacemen
 
 	return res.Returnval, nil
 }
+
+func (l PlacementCompatibilityResult) GetCompatibleDatastores() []types.PbmPlacementHub {
+	var compatibleDatastores []types.PbmPlacementHub
+
+	for _, res := range l {
+		if len(res.Error) == 0 {
+			compatibleDatastores = append(compatibleDatastores, res.Hub)
+		}
+	}
+	return compatibleDatastores
+}
+
+func (l PlacementCompatibilityResult) GetNonCompatibleDatastores() []types.PbmPlacementHub {
+	var nonCompatibleDatastores []types.PbmPlacementHub
+
+	for _, res := range l {
+		if len(res.Error) > 0 {
+			nonCompatibleDatastores = append(nonCompatibleDatastores, res.Hub)
+		}
+	}
+	return nonCompatibleDatastores
+}
+
+func (c *Client) CreatePbmProfile(ctx context.Context, capabilityProfileCreateSpec types.PbmCapabilityProfileCreateSpec) (*types.PbmProfileId, error) {
```

Comment:
Let's also remove "Pbm" from the Client wrapper methods, they are already namespace protected being attached Clieng and in the pbm package.  I would have removed "Pbm" from the generated types too, but was too much trouble wrt to how Go+xml+reflect work.